civis/resources/_resources.py

```diff
@@ -19,9 +19,41 @@
 
 
 API_VERSIONS = ["1.0"]
-BASE_RESOURCES_V1 = ["credentials", "databases", "files", "imports",
-                     "jobs", "models", "predictions", "projects",
-                     "queries", "reports", "scripts", "tables", "users"]
+BASE_RESOURCES_V1 = [
+    'announcements',
+    'apps',
+    'civis',
+    'clusters',
+    'codes',
+    'credentials',
+    'databases',
+    'endpoints',
+    'enhancements',
+    'exports',
+    'files',
+    'groups',
+    'imports',
+    'jobs',
+    'match-targets',
```

Comment:
I'm seeing `match_targets` in my endpoints list at `https://api.civisanalytics.com/endpoints`.  Is that the same for you?

Reply:
yeah, these should use underscores not hyphens.  The CLI-related test should convert them to hyphens (that's what the CLI does to follow UNIX-y norms).  I'll fix this.